k3s-io/k3s
# Add system-default-registry support and remove shared code

#### Proposed Changes ####

* Move registries.yaml handling out to rancher/wharfie
* Add support for --system-default-registry

#### Types of Changes ####

* private registry support

#### Verification ####

* Run standard private registry tests (functionality should be the same, code is just in a different repo)
* Use --system-default registry to point at private registry (may need to also use registries.yaml to trust self-signed certs) - note that all images in `crictl image ls` are explicitly loaded from private registry.

#### Linked Issues ####

For #3280 and #3281

#### Further Comments ####
